### PR TITLE
Aggregate v3 changelog

### LINKED ISSUE #756: v3 changelog
The changelog should be organized with subsections for new/changed datasets for easier reading.

### PR DESCRIPTION
**Pull request by MattF-NSIDC**
## Description

Closes #756 

cc @twilamoon-science 

## Checklist

If an item on this list is done _or not needed_, check it with `[x]` or click the
checkbox.

- [x] The PR description links to issues that it resolves with `closes #{issue_number}`
- [x] Config lockfile updated (`inv config.export > qgreenland/config/cfg-lock.json`)
- [x] Environment lockfile updated if needed (`conda-lock`)
- [x] Version bumped if needed (`bumpversion (major|minor|patch|prerelease|build)`)
- [x] CHANGELOG.md updated (for user-facing changes)
- [x] Documentation updated if needed
- [x] New unit tests if needed

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,35 +1,136 @@
...
+- Update `layer_list.csv` to include new column indicating if each layer is
   stored on disk. Internet-required layers take the value `False`.
-* Add new "Glaciology/Ice sheet velocity" layers:
-  * "GrIMP/Annual ice sheet velocity vectors 2021"
-  * "GrIMP/Annual ice sheet velocity magnitude 2021 (200m)"
-  * "GrIMP/Annual ice sheet velocity x component 2021 (200m)"
-  * "GrIMP/Annual ice sheet velocity y component 2021 (200m)"
-* Improve symbology for "Regional climate models/RACMO model output/Annual mean
+- Establish a more explicit/readable pattern for defining the order of QGreenland's
+  layers and groups in the Layers Panel
```
Comment (trey-stafford) [inline CHANGELOG.md:7]: We removed some other elements from the CHANGELOG that were not 'user' facing. Should we include this? Same w/ the point below about publishing Docker images.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,35 +1,136 @@
...
+  the Raster Circumpolar Arctic Vegetation Map produced by Raynolds et al.,
+  2019.
+
+
+### Updated
+
+- Fix descriptions of "Hydrology" outlets and basins layers.
+- Rename "Places/Place names database" -> "Places/Points of interest". Towns and
+  settlements are now excluded from this layer (in favor of the more-up-to-date
+  "Towns" and "Settlements" layers.
```
Comment (trey-stafford) [inline CHANGELOG.md:61]:
```suggestion
  "Populated places" layer.
```

We merged "Towns" and "Settlements" into one "Populated places" layer.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,35 +1,136 @@
...
+- Update the BedMachine dataset ("Terrain models/BedMachine") to v5, which was
```
Comment (trey-stafford) [inline CHANGELOG.md:62]:
```suggestion
- Update the BedMachine dataset ("Terrain models/BedMachine v5") to v5, which was
```

We now include the version number in the bedmachine layer path.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,35 +1,136 @@
...
+  Chart of the Arctic Ocean (IBCAO) dataset. The GEBCO grid includes the IBCAO
+  grid. This update includes the most recent data and expands the spatial extent
+  of this layer to the QGreenland background boundary to make it consistent with
+  other ocenaography layers.
+- Replace "Human Activity/Research sites/" layers related to PROMICE and GC-Net
+  ("PROMICE automated weather stations", "Former PROMICE automated weather
+  stations", "GC-Net automated weather stations") with one layer ("PROMICE and
+  GC-Net automated weather stations") from a more up-to-date dataset provided by
+  GEUS.
+- Fix title of BedMachine "ice thickness error" layer (remove `/`)
```
Comment (trey-stafford) [inline CHANGELOG.md:95]: Maybe move this up to be next to the other note about updating to BedMachine v5?

**Review by trey-stafford — APPROVED** [review]
Couple of suggested commits but otherwise looks good!

File: CHANGELOG.md (hunk 1 above)
Comment (MattF-NSIDC) [reply CHANGELOG.md:7]: Good point. I agree with removing this one, but think the point about docker images for reproducibility is arguably interesting to users.

File: CHANGELOG.md (hunk 2 above)
Comment (MattF-NSIDC) [reply CHANGELOG.md]: Woops, got it one place but not all the places :laughing:

File: CHANGELOG.md (hunk 4 above)
Comment (MattF-NSIDC) [reply CHANGELOG.md]: b3cc775e

File: CHANGELOG.md (hunk 1 above)
Comment (MattF-NSIDC) [reply CHANGELOG.md]: cb58db57

**Pull request merged**